manifests/config.pp

```diff
@@ -86,12 +86,14 @@
     content => epp('foreman/database.yml.epp', $db_context),
   }
 
+  # Capped at 100 because worst case that's 100 * (5 + 4) = 900 PostgreSQL connections for Katello or 100 * 5 = 500 for vanilla Foreman
```

Comment:
Given our XXL tuning is set to 1000, should we go lower than even 900 connections? (https://github.com/theforeman/foreman-installer/blob/develop/config/foreman.hiera/tuning/sizes/extra-extra-large.yaml#L8) Given we have other parts of the deployment also having to access Postgres.

Reply:
I wasn't quite sure and this was really the nice round number that sort of looked OK. I also think we can increase the XXL tuning above 1000.